gradlew.bat

```diff
@@ -1,89 +1,89 @@
-@rem

```

Comment:
Regarding the .bat files, they were present from the moment i did a rebase from the "main" branch essentially. if you compare the left and right panes there are no changes. So this shouldn't affect the current code, but it is weird that i could not stash/revert these changes after a merge from the  upstream main branch.

Reply:
There are indeed changes in terms of whitespaces. This is addressed by #31398 and you should not include them in your change as they're unrelated to your PR.

Reply:
got it! makes sense thanks for pointing this out 👍🏼